server/Dockerfile

```diff
@@ -10,8 +10,14 @@ ENV INFINISPAN_VERSION 9.4.0.CR3
 # Ensure signals are forwarded to the JVM process correctly for graceful shutdown
 ENV LAUNCH_JBOSS_IN_BACKGROUND true
 
-# Server download location
-ENV DISTRIBUTION_URL https://downloads.jboss.org/infinispan/$INFINISPAN_VERSION/infinispan-server-$INFINISPAN_VERSION.zip
+USER root
+
+# Download server
+ARG DISTRIBUTION_URL=https://downloads.jboss.org/infinispan/$INFINISPAN_VERSION/infinispan-server-$INFINISPAN_VERSION.zip
+ADD $DISTRIBUTION_URL /tmp
```

Comment:
This instruction here makes the image ~200Mb bigger. We need to do all file extracting, copying and moving operations in a single RUN command, as it is on  [master](https://github.com/jboss-dockerfiles/infinispan/blob/master/server/Dockerfile#L32)